cli/cmd/root.go

```diff
@@ -77,16 +77,19 @@ func runCmd(ctx context.Context, ver config.Version) error {
 	rootCmd.AddCommand(completionCmd)
 	rootCmd.AddCommand(versioncmd.VersionCmd())
 
+	cmd := auth.AuthCmd(cfg)
+	cmd.PersistentFlags().StringVar(&cfg.AdminURL, "api-url", "https://admin.rilldata.com", "Base URL for the admin API")
+	rootCmd.AddCommand(cmd)
+
 	// Add sub-commands for admin
 	// (This allows us to add persistent flags that apply only to the admin-related commands.)
 	adminCmds := []*cobra.Command{
-		auth.AuthCmd(cfg),
 		org.OrgCmd(cfg),
 		project.ProjectCmd(cfg),
 	}
 	for _, cmd := range adminCmds {
 		cmd.PersistentFlags().StringVar(&cfg.AdminURL, "api-url", "https://admin.rilldata.com", "Base URL for the admin API")
-		cmd.PersistentFlags().StringVar(&cfg.AdminToken, "api-token", "", "Token for authenticating with the admin API")
+		cmd.PersistentFlags().StringVar(&cfg.AdminToken, "api-token", token, "Token for authenticating with the admin API")
```

Comment:
This will cause the token to be printed in the help menu with `-h`. Probably we separate fields in config for this, one for the default and one for override (and then a getter func on `cfg` for picking the right one) – like Rakesh did here: https://github.com/rilldata/rill-developer/pull/1845#discussion_r1128654075